src/app_service/service/wallet_account/service.nim

```diff
@@ -999,7 +1019,7 @@ QtObject:
     else:
       if self.walletAccountsContainsAddress(account.address):
         self.updateAccountInLocalStoreAndNotify(account.address, account.name, account.colorId, account.emoji,
-          none(bool), notify)
+          none(bool), prodPreferredChains = "", testPreferredChains = "", notify)
```

Comment:
@Khushboo-dev-cpp if preferred chains change for an address is coming from syncing, then we surely need to update them on the synced side, no?

Reply:
True, I'll update these values too then!